notebooks/03.09-Pivot-Tables.ipynb

```diff
@@ -49,9 +49,7 @@
   {
```

Comment:
<p>Test comment</p>

 _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
v2

Reply:
v3

Reply:
v4

Reply:
<p>ore about this data, we might begin by grouping according to gender, survival status, or some combination thereof. If you have read t</p>